source/hardware/controllers/hercules_djcontrol_jogvision.rst

```diff
@@ -0,0 +1,138 @@
+Hercules DJControl Jogvision
+============================
+
+-  `Manufacturer's product page <https://www.hercules.com/en-us/product/djcontroljogvision-2old/>`__
+-  `Manufacturer's support and downloads page <https://support.hercules.com/en/product/djcontroljogvision-en/>`__
+-  `Forum thread <https://www.mixxx.org/forums/viewtopic.php?f=7&t=12580>`__
+  
+Whether you’re just getting started DJing or you’ve already refined your skills, you can let your creativity run wild.
+Enjoy excellent precision and comfort to create your mixes using the circular displays on the jog wheels to guide your movements and perfect your scratches,
+and have fun with the amazing contactless AIR Control sensor.
+Your adventure starts right here and now. It’s your turn\!
+
+Compatibility
+-------------
+
+This controller is a class compliant USB MIDI and audio device, so it
+can be used without any special drivers on GNU/Linux, Mac OS X, and
+Windows. However, if you wish to use the `ASIO sound
+API <https://mixxx.org/manual/latest/en/chapters/preferences.html?highlight=asio#windows>`__
+under Windows, please install the latest driver package available from
+the `Support
+page <https://support.hercules.com/en/product/djcontrolinpulse300-en//>`__.
+
+Sound card setup
+----------------
+
+This controller has built-in 4 channel output sound card, with:
+
+- OUTPUTS:
+
+  - MASTER:RCA and 3.5mm jack
+  - HEADPHONE: 6.35mm and 3.5mm jack
+  - BOOTH: RCA and 3.5mm jack
+- INPUTS:
+
+  - MIC:6.35mm jack
+  - AUX: 3.5mm jack
+
+- Open **Preferences \> Sound Hardware**  
+- Select the **Output** tab  
+- From the **Master** drop-down menu, select the audio interface, then **Channels 1-2**  
+- From the **Headphones** drop-down menu, select the audio interface, then **Channels 3-4**  
+- Click **Apply** to save the changes.  
+
+Please refer to `the user manual <https://mixxx.org/manual/latest/en/chapters/example_setups.html#laptop-and-external-usb-audio-interface>`__
+for more details about the audio configuration in Mixxx.
+
+.. note::
+   The **Master** and **Headphone** knobs, as well as the
+   **Master** button are hardware controls and interact directly with the
+   integrated sound card’s output.
+   
+Mapping description
+-------------------
+
+Save both MIDI and script files to your `user controller mapping
```

Comment:
I don't think this paragraph is needed and could confuse users. We'll only merge this when the mapping PR has been merged as well. In that case, the mapping is built-in and it suffices to select the correct mapping from the drop down menu.